### PR TITLE
[#621] Additional custom fields for voucher in jira

### LINKED ISSUE #621: Addition of three additional fields in JIRA (connected with vouchers functionality)
1. We need to request to add three additional fields conected with voucher to the MP and next do the final mapping of them.
@agpul 

In order to complete task connected with Helix Nebula and vouchers please insert following changes:

    Add three new custom fields:
        CP-Platforms (string, optional)
        CP-INeedAVoucher (string, optional)
        CP-VoucherID (string, optional)

2. This 3  attributes should be propagated to the JIRA tickets for service order @michal-szostak

### PR DESCRIPTION
**Pull request by martaswiatkowska**
Tree additional fields for voucher added to jira 
    CP-Platforms (string, optional)
    CP-INeedAVoucher (boolean, optional)
    CP-VoucherID (string, optional)

Fix #621

### CODE REVIEW

**Comment by kmarszalek** [comment]
changelog?

File: app/models/jira/client.rb [hunk 1]
```diff
@@ -141,6 +144,12 @@ def generate_custom_field_value(field_name, project_item)
       project_item.project_name
     when "CP-UserGroupName"
       project_item.user_group_name
+    when "CP-Platforms"
+      project_item.offer.service.platforms.map(&:name).join(", ") || nil
+    when "CP-INeedAVoucher"
+      "#{project_item.request_voucher}"
```
Comment (michal-szostak) [inline app/models/jira/client.rb]: In the end `request_voucher` field in `ProjectItem` has been changed to `voucherable`

**Comment by agpul** [comment]
I think that I'm not able to test it before merge changes to the marketplace-docker because JIRA isn't connected with PRs

File: app/models/jira/client.rb (hunk 1 above)
Comment (martaswiatkowska) [reply app/models/jira/client.rb]: Only `Offer` has field `voucherable`. `ProjectItem` has `request_voucher` and method `voucherable?`

**Pull request merged**